<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MMOD Degtyarev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QTabWidget" name="mainTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="brvTab">
     <attribute name="title">
      <string>BRV</string>
     </attribute>
     <widget class="Line" name="BRVLine">
      <property name="geometry">
       <rect>
        <x>396</x>
        <y>0</y>
        <width>8</width>
        <height>572</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>371</width>
        <height>577</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="BRVMCGVBox">
       <item>
        <layout class="QHBoxLayout" name="BRVMCGSeedHBox">
         <item>
          <widget class="QLabel" name="BRVMCGSeedLabel">
           <property name="text">
            <string>Seed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMCGSeedEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMCGMultiplierHBox">
         <item>
          <widget class="QLabel" name="BRVMCGMultiplierLabel">
           <property name="text">
            <string>Multiplier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMCGMultiplierEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>1664525</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMCGModulusHBox">
         <item>
          <widget class="QLabel" name="BRVMCGModulusLabel">
           <property name="text">
            <string>Modulus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMCGModulusEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>4294967296</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMCGIncrementHBox">
         <item>
          <widget class="QLabel" name="BRVMCGIncrementLabel">
           <property name="text">
            <string>Increment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMCGIncrementEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>1013904223</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMCGCountHBox">
         <item>
          <widget class="QLabel" name="BRVMCGCountLabel">
           <property name="text">
            <string>Count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMCGCountEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="BRVMCGStatusLabel">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMCGActionHBox">
         <item>
          <widget class="QPushButton" name="BRVMCGClearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BRVMCGGenerateButton">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="BRVMCGGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="title">
          <string>Congruential Generator</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="BRVMSVBox">
       <item>
        <layout class="QHBoxLayout" name="BRVMSSeedHBox">
         <item>
          <widget class="QLabel" name="BRVMSSeedLabel">
           <property name="text">
            <string>Seed number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMSSeedEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMSCountHBox">
         <item>
          <widget class="QLabel" name="BRVMSCountLabel">
           <property name="text">
            <string>Count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BRVMSCountEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="BRVMSStatus">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BRVMSActionHBox">
         <item>
          <widget class="QPushButton" name="BRVMSClearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BRVMSGenerateButton">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="BRVMSGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="title">
          <string>Middle-Square</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>BRVLine</zorder>
    </widget>
    <widget class="QWidget" name="crvTab">
     <attribute name="title">
      <string>CRV</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="CRVVBox">
       <item>
        <layout class="QHBoxLayout" name="CRVAHBox">
         <item>
          <widget class="QLabel" name="CRVALabel">
           <property name="text">
            <string>Param (a)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CRVAEdit">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="CRVBHBox">
         <item>
          <widget class="QLabel" name="CRVBLabel">
           <property name="text">
            <string>Param (b)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CRVBEdit">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="CRVCountHBox">
         <item>
          <widget class="QLabel" name="CRVCountLabel">
           <property name="text">
            <string>Count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CRVCountEdit">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="CRVActionHBox">
         <item>
          <widget class="QPushButton" name="CRVClearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CRVGenerateButton">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="CRVGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="title">
          <string>CRV Pareto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="CRVStatusLabel">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CRVStatusText">
         <property name="text">
          <string>StatusText</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="modelTab">
     <attribute name="title">
      <string>Model</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
